dashboard/app/views/levels/_external.html.haml

```diff
@@ -31,8 +31,12 @@
 
 - if video && use_large_video_player
   .clear
-  .video-container.external-video-container
-  %a.btn.btn-large.btn-primary.next-stage.submitButton.pull-right= t('continue')
+  .video-container.external-video-container{{style: 'max-width: 970px; margin: auto'}}
```

Comment:
I'd prefer not to use in-line styling, but wasn't sure where I could put this, especially if we don't want it to affect styles of other uses of these elements.

Reply:
what you have here seems good to me, given that it is a small amount of CSS. the alternative is to put this in application.scss, and scope it by limiting it to elements which match `.video-container.external-video-container`. breaking up our CSS is something on my list for optimizing our page load times, although it's getting a bit far down the list at this point.

Reply:
application.scss is kind of a beast - I see what you mean. Ok, I'll leave this here then.